### DISCUSSION TITLE
Welcome to redis-operator Discussions!

### DISCUSSION THREAD

**Discussion by iamabhishek-dubey** [OT-CONTAINER-KIT/redis-operator#64] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaiTest** [comment]
i need to mount a volume from the Redis pod for mounting /data to a PV,
how to add this to the pod config?
